libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -426,6 +430,14 @@ class MapboxNavigation(
         navigator.resetRideSession()
     }
 
+    /**
+     * Query if the [MapboxNavigation] is running a foreground service
+     * @return true if the [MapboxNavigation] is running a foreground service else false
+     */
+    fun isTripSessionRunningService(): Boolean {
```

Comment:
```suggestion
    fun isRunningForegroundService(): Boolean {
```
nit: this probably better describes the information carried by the flag.